include/triton-shared/Conversion/TritonArithToLinalg/ConversionPatterns.hpp

```diff
@@ -1741,6 +1741,10 @@ struct DenseConstantConverter : public OpConversionPattern<arith::ConstantOp> {
 class CumSumConverter : public OpConversionPattern<triton::ScanOp> {
   using OpConversionPattern<triton::ScanOp>::OpConversionPattern;
 
+  bool isAddOp(Operation *op) const {
```

Comment:
I would just remove this function and use `isa` directly in the code below (better readability) especially since it is quite simple.

Reply:
thanks! i simplified the code, looks much better now